README.rst

```diff
@@ -151,6 +153,12 @@ The repository consists mostly of externally hosted subrepositories:
 .. |libtarver| replace:: 1.2.20 + patches (master)
 .. _libtarver: https://github.com/tklauser/libtar/commit/6379b5d2ae777dad576aeae70566740670057821
 
+.. _cfitsio: https://github.com/healpy/cfitsio.git
+.. |cfitsiolic| replace:: MIT license
```

Comment:
I've inferred that from what was said during last meeting, although I am not sure if it is factually correct since license itself is quite ambiguous.